Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN go install github.com/mozilla/tls-observatory/tlsobs-api && \
     go install github.com/mozilla/tls-observatory/tlsobs-scanner && \
     go install github.com/mozilla/tls-observatory/tlsobs-runner && \
     apt-get update -y && \
-    apt-get install git libcurl4-nss-dev libnss3 libnss3-dev clang -y && \
+    apt-get install git libcurl4-nss-dev libnss3 libnss3-dev clang postgresql-client -y && \
```

Comment:
Needed a way to automate creation of the database and schema from the container. fwiw this doesn't increase the image size substantially. `docker images` shows a size of 1.34GB before this, and the same size after the change.